### PR TITLE
Make init return error instead of fail

### PR DESCRIPTION
**Pull request by dkolistratova**
#440

### CODE REVIEW

File: low/low.go [hunk 1]
```diff
@@ -635,9 +641,13 @@ func ReportMempoolsState() {
 }
 
 // CreateKni creates a KNI device
-func CreateKni(portId uint16, core uint, name string) {
+func CreateKni(portId uint16, core uint, name string) error {
 	mempool := (*C.struct_rte_mempool)(CreateMempool("KNI"))
-	C.create_kni(C.uint16_t(portId), C.uint32_t(core), C.CString(name), mempool)
+	ret := C.create_kni(C.uint16_t(portId), C.uint32_t(core), C.CString(name), mempool)
```
Comment (ifilippov) [inline low/low.go]: either use ret variable in returned error or remove it

File: low/low_test.go [hunk 2]
```diff
@@ -9,10 +9,12 @@ import "testing"
 import "math/rand"
 import "time"
 
-func init() {
+func TestInit(t *testing.T) {
```
Comment (ifilippov) [inline low/low_test.go]: Why?

Comment (dkolistratova) [reply low/low_test.go]: to use t.FatalF

Comment (ifilippov) [reply low/low_test.go]: Bad practice. If it wasn't initialized test will also fail.
In you case what if user use go test -test=TestReorder ?

Comment (ifilippov) [reply low/low_test.go]: use log.Fatal if you want. Or nothing.

**Comment by ifilippov** [comment]
Add checking to kni.go example. Than change low test and you can commit.

**Comment by dkolistratova** [comment]
kni example already has this check cause CreateKniDevice returned error before

**Pull request merged**